src/vmm/src/cpu_config/x86_64/cpuid/normalize.rs

```diff
@@ -424,6 +437,15 @@ impl super::Cpuid {
 
         Ok(())
     }
+
+    // Update extended cache features entry
+    fn update_extended_cache_features(&mut self) -> Result<(), ExtendedCacheFeaturesError> {
+        let guest_leaf_0x80000006 = self
+            .get_mut(&CpuidKey::leaf(0x80000006))
+            .ok_or(ExtendedCacheFeaturesError::MissingLeaf0x80000006)?;
+        guest_leaf_0x80000006.result = cpuid(0x80000006).into();
```

Comment:
Should we also apply this fix patch like what the kernel does?
https://github.com/amazonlinux/linux/commit/eeb69eab57c6604ac90b3fd8e5ac43f24a5535b1
https://lore.kernel.org/all/20220929225203.2234702-2-jmattson@google.com/